Recategorize integrations

jen-huang opened the pull request:
## What does this PR do?

This PR adjusts integration categories based on the work done in https://github.com/elastic/ingest-dev/issues/1349 to streamline the UX for browsing integrations and introduces subcategories for particularly overloaded top-level categories.

All updated integrations have their patch version bumped.

**Reviewers:** Thank you for taking the time to review. I know this is a big PR! I recommend:

1. You filter to files that you own
2. If there is disagreement about any of the categories, _please discuss offline_ and get consensus from the product owner (@mukeshelastic can help) before requesting a change

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have added an entry to my package's `changelog.yml` file.

## Related PRs
- [x] https://github.com/elastic/package-registry/pull/970
- [x] https://github.com/elastic/package-spec/pull/490
- [x] https://github.com/elastic/kibana/pull/153221
- [ ] https://github.com/elastic/kibana/pull/153216
- [ ] https://github.com/elastic/endpoint-package/pull/354

## Related issues

- https://github.com/elastic/kibana/issues/147125
- https://github.com/elastic/ingest-dev/issues/1349
- https://github.com/elastic/package-spec/issues/451

elasticmachine commented:
## :broken_heart: Build Failed


<!-- BUILD BADGES-->
> _the below badges are clickable and redirect to their specific view in the CI or DOCS_
[![Pipeline View](https://img.shields.io/badge/pipeline-pipeline%20-green)](https://fleet-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations%2FPR-5123/detail/PR-5123/1//pipeline) [![Test View](https://img.shields.io/badge/test-test-green)](https://fleet-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations%2FPR-5123/detail/PR-5123/1//tests) [![Changes](https://img.shields.io/badge/changes-changes-green)](https://fleet-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations%2FPR-5123/detail/PR-5123/1//changes) [![Artifacts](https://img.shields.io/badge/artifacts-artifacts-yellow)](https://fleet-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations%2FPR-5123/detail/PR-5123/1//artifacts) [![preview](https://img.shields.io/badge/docs-preview-yellowgreen)](http://integrations_5123.docs-preview.app.elstc.co/diff)  [![preview](https://img.shields.io/badge/elastic-observability-blue)](https://ci-stats.elastic.co/app/apm/services/fleet-ci/transactions/view?rangeFrom=2023-01-27T00:28:16.611Z&rangeTo=2023-01-27T00:48:16.611Z&transactionName=BUILD+Ingest-manager%2Fintegrations%2FPR-%7Bnumber%7D&transactionType=job&latencyAggregationType=avg&traceId=fc2de96f2409c4e5051d07e10d3c13d7&transactionId=323db0d72f5a5d6e)

<!-- BUILD SUMMARY-->
<details><summary>Expand to view the summary</summary>
<p>

#### Build stats




* Start Time: 2023-01-27T00:38:16.611+0000


* Duration: 11 min 16 sec





</p>
</details>

<!-- TEST RESULTS IF ANY-->


<!-- STEPS ERRORS IF ANY -->














### Steps errors [![18](https://img.shields.io/badge/18%20-red)](https://fleet-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations%2FPR-5123/detail/PR-5123/1//pipeline)
  <details><summary>Expand to view the steps failures</summary>
  <p>


> Show only the first 10 steps failures

  
  
  
  
##### `Check integration: auth0`
<ul>
<li>Took 0 min 1 sec . View more details <a href="https://fleet-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Ingest-manager/pipelines/integrations/pipelines/PR-5123/runs/1/steps/1583/log/?start=0">here</a></li>
<li>Description: <code>../../build/elastic-package check -v</code></l1>
</ul>
  
  
  
  
##### `Check integration: aws`
<ul>
<li>Took 0 min 4 sec . View more details <a href="https://fleet-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Ingest-manager/pipelines/integrations/pipelines/PR-5123/runs/1/steps/1572/log/?start=0">here</a></li>
<li>Description: <code>../../build/elastic-package check -v</code></l1>
</ul>
  
  
  
  
##### `Check integration: awsfargate`
<ul>
<li>Took 0 min 1 sec . View more details <a href="https://fleet-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Ingest-manager/pipelines/integrations/pipelines/PR-5123/runs/1/steps/1595/log/?start=0">here</a></li>
<li>Description: <code>../../build/elastic-package check -v</code></l1>
</ul>
  
  
  
  
##### `Check integration: azure`
<ul>
<li>Took 0 min 1 sec . View more details <a href="https://fleet-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Ingest-manager/pipelines/integrations/pipelines/PR-5123/runs/1/steps/1848/log/?start=0">here</a></li>
<li>Description: <code>../../build/elastic-package check -v</code></l1>
</ul>
  
  
  
  
##### `Check integration: azure_billing`
<ul>
<li>Took 0 min 0 sec . View more details <a href="https://fleet-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Ingest-manager/pipelines/integrations/pipelines/PR-5123/runs/1/steps/1803/log/?start=0">here</a></li>
<li>Description: <code>../../build/elastic-package check -v</code></l1>
</ul>
  
  
  
  
##### `Check integration: azure_metrics`
<ul>
<li>Took 0 min 1 sec . View more details <a href="https://fleet-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Ingest-manager/pipelines/integrations/pipelines/PR-5123/runs/1/steps/1747/log/?start=0">here</a></li>
<li>Description: <code>../../build/elastic-package check -v</code></l1>
</ul>
  
  
  
  
##### `Check integration: netscout`
<ul>
<li>Took 0 min 1 sec . View more details <a href="https://fleet-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Ingest-manager/pipelines/integrations/pipelines/PR-5123/runs/1/steps/1763/log/?start=0">here</a></li>
<li>Description: <code>../../build/elastic-package check -v</code></l1>
</ul>
  
  
  
  
##### `Check integration: tomcat`
<ul>
<li>Took 0 min 1 sec . View more details <a href="https://fleet-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Ingest-manager/pipelines/integrations/pipelines/PR-5123/runs/1/steps/1901/log/?start=0">here</a></li>
<li>Description: <code>../../build/elastic-package check -v</code></l1>
</ul>
  
  
  
  
##### `Google Storage Download`
<ul>
<li>Took 0 min 0 sec . View more details <a href="https://fleet-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Ingest-manager/pipelines/integrations/pipelines/PR-5123/runs/1/steps/2132/log/?start=0">here</a></li>

</ul>
  
  
  
  
##### `Google Storage Download`
<ul>
<li>Took 0 min 0 sec . View more details <a href="https://fleet-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Ingest-manager/pipelines/integrations/pipelines/PR-5123/runs/1/steps/2145/log/?start=0">here</a></li>

</ul>
  
  </p>
  </details>



## :robot: GitHub comments

<details><summary>Expand to view the GitHub comments</summary>
<p>

To re-run your PR in the CI, just comment with:



- `/test` : Re-trigger the build.




</p>
</details>

<!--COMMENT_GENERATED_WITH_ID_comment.id-->

elasticmachine commented:
### :globe_with_meridians: Coverage report

Name | Metrics % (`covered/total`) | Diff
 --- | --- | ---
Packages | 100.0% (`5/5`) | :green_heart: 
Files | 100.0% (`5/5`) | :green_heart: 
Classes | 100.0% (`5/5`) | :green_heart: 
Methods | 74.51% (`38/51`) | :-1: -25.49
Lines | 84.962% (`113/133`) | :-1: -9.543
Conditionals | 100.0% (`0/0`) | :green_heart: 

<!--COMMENT_GENERATED_WITH_ID_coverage-->

jsoriano commented:
@jen-huang this can be rebased, missing categories should be available now after https://github.com/elastic/integrations/pull/5144.

jsoriano commented on packages/snyk/manifest.yml:
```diff
@@ -1,12 +1,13 @@
 type: integration
 categories:
   - security
+  - vulnerability_management
```
This category hasn't been added yet. It will need a new release of package-registry and package-spec, it can still take some days/weeks.

jsoriano commented on packages/tenable_sc/manifest.yml:
```diff
@@ -2,13 +2,14 @@ format_version: 1.0.0
 type: integration
 categories:
   - security
+  - vulnerability_management
```
Same here about category not released yet.

endorama commented on packages/gcp/manifest.yml:
```diff
@@ -149,6 +153,8 @@ policy_templates:
     title: Google Cloud Platform (GCP) Load Balancing metrics
     description: Collect Load Balancing metrics from Google Cloud Platform (GCP) with Elastic Agent
+    categories:
+      - security
```
@jen-huang `gcp.loadbalancing` collects also metrics, I would add the `infrastructure` category as well.

jen-huang replied:
@endorama Is "Security" even accurate for this? Should it just be swapped out for Infrastructure?

endorama replied:
Security is correct as logs are more related to security posture than observability.
In this case we have both logs and metrics, which makes this integration useful for both purposes.

endorama reviewed (commented):
Reviewing `gcp` package for Observability, I left a comment.

CohenIdo commented on packages/cloud_security_posture/manifest.yml:
```diff
@@ -1,19 +1,15 @@
 type: integration
 categories:
-  - containers
+  - infrastructure
```
hey @jen-huang May we add "cloud security" tag or something that will describe our integration better?

@oren-zohar @kfirpeled @eyalkraft

jen-huang replied:
This should be discussed with @mukeshelastic and opened as a separate proposal for a new category. If approved, this package can subsequently be updated.

oren-zohar replied:
ok no problem then, cc @tinnytintin10

jamiehynds replied:
@tinnytintin10 can you confirm that you're ok with categorization for the Cloud Security Posture package? Main category set to `security` with `cloud` and `aws` also applied. `infrastructure` category will be dropped. Any additional keywords you'd like to specify, so when a user searches on the integrations page the CSP integration will appear?

tinnytintin10 replied:
Looks like the KSPM policy template still has the `container` categorization. With this in mind, I am okay with removing the container & and infrastructure categorizations from the main integration package 👍🏾

jamiehynds commented on packages/snyk/manifest.yml:
```diff
@@ -1,13 +1,12 @@
 type: integration
 categories:
   - security
-  - vulnerability_management
```
@jen-huang why was the vulnerability management category removed? We currently have AWS Inspector, Tenable.sc, Tenable,io and Snyk which are a great fit. We are currently working on other vulnerability management integrations including Qualys and Rapid7.

jsoriano replied:
It was missed on the first iteration of new categories, and is still not supported by the package registry and the package spec (https://github.com/elastic/package-spec/pull/471). Changes are merged but not released yet.

SubhrataK commented:
Hi,

Here are the changes to be made to the Observability Integrations:

1. **Infrastructure** Category is dropped. So all the integrations are now mapped to **Observability** in Category 1 starting from Row no. 158 (APM Android Agent) to Row no. 304 (Zookeeper Metrics)in the spreadsheet. Exceptions are: MySQL Enterprise and Sample Data
2. New Categories added are: APM, Compute, Kubernetes and Message Queues.
3. New Integrations added are: 

<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>

<meta http-equiv="Content-Style-Type" content="text/css">

</head>
<body>


Integration | Owner | Category 1 | Category 2 | Category 3 | Subcategory (if applicable, open ended) | Keywords (additional terms for search)
-- | -- | -- | -- | -- | -- | --
APM Android Agent | Observability | Observability | APM |   |   | APM
APM Go Agent | Observability | Observability | APM |   |   | APM
APM Java Agent | Observability | Observability | APM |   |   | APM
APM .Net Agent | Observability | Observability | APM |   |   | APM
APM iOS Agent | Observability | Observability | APM |   |   | APM
APM Node.js Agent | Observability | Observability | APM |   |   | APM
APM PHP Agent | Observability | Observability | APM |   |   | APM
APM Python Agent | Observability | Observability | APM |   |   | APM
APM Ruby Agent | Observability | Observability | APM |   |   | APM
APM RUM(JS) Agent | Observability | Observability | APM |   |   | APM
APM Django Agent | Observability | Observability | APM |   |   | APM
APM Flask Agent | Observability | Observability | APM |   |   | APM
APM Rack Agent | Observability | Observability | APM |   |   | APM
APM OpenTelemetry Agent | Observability | Observability | APM |   |   | APM
Airflow | Observability | Observability |   |   |   |  
Amazon CloudFront | Observability | Observability | AWS |   | Observability: Monitoring |  
Amazon VPN | Observability | Observability | AWS | Network |   |  
Azure Application Insights Metrics | Observability |   | Observability | Azure |   |  
Azure Application State Insights Metrics | Observability |   | Observability | Azure |   |  
Citrix ADC | Observability |   | Observability | Network |   |  
Graphite | Observability |   | Observability |   |   |  
Jolokia | Observability |   | Observability |   |   |  


</body>
</html>

4. Updated Integrations : All the integrations to be updated to Observability as Category 1 and the below listed integrations have Category 2/3 also updated :

<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>

<meta http-equiv="Content-Style-Type" content="text/css">

</head>
<body>


Integration | Owner | Category 1 | Category 2 | Category 3 | Subcategory (if applicable, open ended) | Keywords (additional terms for search)
-- | -- | -- | -- | -- | -- | --
ActiveMQ Logs | Observability | Observability | Message Queues |   |   |  
ActiveMQ Metrics | Observability | Observability | Message Queues |   |   |  
Amazon EC2 | Observability | Observability | AWS | Compute |   |  
Amazon ECS | Observability | Observability | AWS | Containers |   |  
Amazon SQS | Observability | Observability | AWS | Message Queues |   | message queue
AWS Fargate | Observability | Observability | AWS | Compute | Observability: Serverless | Container as a Service, CaaS
AWS Lambda | Observability | Observability | AWS | Compute | Observability: Serverless | FaaS
Azure Application Insights Metrics | Observability | Observability | Azure |   |   |  
Azure Application State Insights Metrics | Observability | Observability | Azure |   |   |  
Azure Container Instance Metrics | Observability | Observability | Azure | Containers |   |  
Azure Container Registry Metrics | Observability | Observability | Azure | Containers |   |  
Azure Container Service Metrics | Observability | Observability | Azure | Containers |   |  
Google Cloud Platform (GCP) PubSub metrics | Observability | Observability | Google cloud | Message Queues |   | Messaging
IBM MQ Logs | Observability | Observability | Message Queues |   |   |  
IBM MQ Metrics | Observability | Observability | Message Queues |   |   |  
Kubernetes | Observability | Observability | Containers | Kubernetes | Observability: Kubernetes |  
Nginx Ingress Controller Logs | Observability | Observability | Containers | Kubernetes | Observability: Kubernetes | Ingress, Kubernetes
RabbitMQ Logs | Observability | Observability | Message Queues |   | Observability: Message Broker |  
Salesforce | Observability | Observability |   |   |   | CRM


</body>
</html>

jen-huang commented:
@SubhrataK Thanks for the summary of requested changes. A few notes:

1. Even if we do not tag any integrations with `Infrastructure`, it is still the parent category of many subcategories as defined in [the list of categories](https://docs.google.com/spreadsheets/d/1NUR2ypKSjtHvrAq1MVK31SkVnzh7vQN8TptnwsayQmI/edit#gid=1476422042). This means that a category called `Infrastructure` will still appear in Integrations UI and will populate with integrations tagged with any of its assigned subcategories. Is this expected for you?
2. New categories will need to be requested outside of this PR. The source of truth for approved & available categories lives in [this file](https://github.com/elastic/package-registry/blob/main/categories/categories.yml). Please open a proposal issue in the [package-spec](https://github.com/elastic/package-spec) repo for the new categories APM, Compute, and Message Queues (Kubernetes already exists). The integrations can then be updated once the categories are added.
3. What are the APM integrations? I do not see them anywhere today.

SubhrataK commented:
Hi @jen-huang,

Please find the answers below:

1. As discussed, please remove the Infrastructure category from the Subcategories too. The parent category would be "Observability"
2. As suggested, I have raised a proposal issue [https://github.com/elastic/package-spec/issues/488](), for adding new categories. Please note that the new categories to be added are APM and Message Queues only (discarding Compute as there are not many integrations that fall under compute)
3. The APM list that I added is available here: [https://www.elastic.co/integrations/data-integrations?solution=all-solutions&category=languages](), basically all the integrations which support traces. But I did not find them in the other  integrations catalog available here: [https://cloud.elastic.co/](). Instead all these are available as different tabs under Elastic APM. Please hold on to mapping of these integrations. We will sort this out and will take it up in the next iteration later.

jen-huang commented:
@SubhrataK Thanks for answering. I've updated this PR with the requested changes (as much as I can do for now). The PR description is also updated with related PRs in other repos.

Going to go ahead and merge this. There are two test failures for `nats` and `stan` packages that _might_ be due to Go version upgrade. These packages haven't been updated in a while. Will check to see if anything should be done here.

elasticmachine commented:
Package 1password - 1.8.2 containing this change is available at https://epr.elastic.co/search?package=1password

elasticmachine commented:
Package activemq - 0.6.1 containing this change is available at https://epr.elastic.co/search?package=activemq

elasticmachine commented:
Package airflow - 0.0.3 containing this change is available at https://epr.elastic.co/search?package=airflow

elasticmachine commented:
Package akamai - 2.4.1 containing this change is available at https://epr.elastic.co/search?package=akamai

elasticmachine commented:
Package apache - 1.8.1 containing this change is available at https://epr.elastic.co/search?package=apache

elasticmachine commented:
Package apache_spark - 0.4.1 containing this change is available at https://epr.elastic.co/search?package=apache_spark

elasticmachine commented:
Package atlassian_bitbucket - 1.6.1 containing this change is available at https://epr.elastic.co/search?package=atlassian_bitbucket

elasticmachine commented:
Package atlassian_confluence - 1.7.1 containing this change is available at https://epr.elastic.co/search?package=atlassian_confluence

elasticmachine commented:
Package atlassian_jira - 1.7.1 containing this change is available at https://epr.elastic.co/search?package=atlassian_jira

elasticmachine commented:
Package auditd - 3.5.1 containing this change is available at https://epr.elastic.co/search?package=auditd

elasticmachine commented:
Package auditd_manager - 1.6.1 containing this change is available at https://epr.elastic.co/search?package=auditd_manager

elasticmachine commented:
Package auth0 - 1.4.1 containing this change is available at https://epr.elastic.co/search?package=auth0

elasticmachine commented:
Package aws - 1.32.1 containing this change is available at https://epr.elastic.co/search?package=aws

elasticmachine commented:
Package aws_logs - 0.3.3 containing this change is available at https://epr.elastic.co/search?package=aws_logs

elasticmachine commented:
Package awsfargate - 0.2.1 containing this change is available at https://epr.elastic.co/search?package=awsfargate

elasticmachine commented:
Package azure - 1.5.12 containing this change is available at https://epr.elastic.co/search?package=azure

elasticmachine commented:
Package azure_application_insights - 1.0.5 containing this change is available at https://epr.elastic.co/search?package=azure_application_insights

elasticmachine commented:
Package azure_billing - 1.1.2 containing this change is available at https://epr.elastic.co/search?package=azure_billing

elasticmachine commented:
Package azure_metrics - 1.0.16 containing this change is available at https://epr.elastic.co/search?package=azure_metrics

elasticmachine commented:
Package barracuda - 0.13.1 containing this change is available at https://epr.elastic.co/search?package=barracuda

elasticmachine commented:
Package barracuda_cloudgen_firewall - 0.3.1 containing this change is available at https://epr.elastic.co/search?package=barracuda_cloudgen_firewall

elasticmachine commented:
Package bluecoat - 0.12.1 containing this change is available at https://epr.elastic.co/search?package=bluecoat

elasticmachine commented:
Package box_events - 0.4.1 containing this change is available at https://epr.elastic.co/search?package=box_events

elasticmachine commented:
Package carbon_black_cloud - 1.6.1 containing this change is available at https://epr.elastic.co/search?package=carbon_black_cloud

elasticmachine commented:
Package carbonblack_edr - 1.7.1 containing this change is available at https://epr.elastic.co/search?package=carbonblack_edr

elasticmachine commented:
Package cassandra - 1.4.1 containing this change is available at https://epr.elastic.co/search?package=cassandra

elasticmachine commented:
Package cef - 2.6.2 containing this change is available at https://epr.elastic.co/search?package=cef

elasticmachine commented:
Package checkpoint - 1.16.1 containing this change is available at https://epr.elastic.co/search?package=checkpoint

elasticmachine commented:
Package cisco_aironet - 0.3.1 containing this change is available at https://epr.elastic.co/search?package=cisco_aironet

elasticmachine commented:
Package cisco_asa - 2.14.1 containing this change is available at https://epr.elastic.co/search?package=cisco_asa

elasticmachine commented:
Package cisco_duo - 1.8.1 containing this change is available at https://epr.elastic.co/search?package=cisco_duo

elasticmachine commented:
Package cisco_ftd - 2.9.1 containing this change is available at https://epr.elastic.co/search?package=cisco_ftd

elasticmachine commented:
Package cisco_ise - 1.6.1 containing this change is available at https://epr.elastic.co/search?package=cisco_ise

elasticmachine commented:
Package cisco_secure_email_gateway - 1.6.1 containing this change is available at https://epr.elastic.co/search?package=cisco_secure_email_gateway

elasticmachine commented:
Package cisco_secure_endpoint - 2.8.1 containing this change is available at https://epr.elastic.co/search?package=cisco_secure_endpoint

elasticmachine commented:
Package cisco_umbrella - 1.6.1 containing this change is available at https://epr.elastic.co/search?package=cisco_umbrella

elasticmachine commented:
Package citrix_adc - 0.5.1 containing this change is available at https://epr.elastic.co/search?package=citrix_adc

elasticmachine commented:
Package citrix_waf - 1.3.1 containing this change is available at https://epr.elastic.co/search?package=citrix_waf

elasticmachine commented:
Package cloud_security_posture - 1.3.0-preview2 containing this change is available at https://epr.elastic.co/search?package=cloud_security_posture

elasticmachine commented:
Package cloudflare - 2.4.2 containing this change is available at https://epr.elastic.co/search?package=cloudflare

elasticmachine commented:
Package cloudflare_logpush - 0.5.1 containing this change is available at https://epr.elastic.co/search?package=cloudflare_logpush

elasticmachine commented:
Package cockroachdb - 1.1.1 containing this change is available at https://epr.elastic.co/search?package=cockroachdb

elasticmachine commented:
Package containerd - 0.2.2 containing this change is available at https://epr.elastic.co/search?package=containerd

elasticmachine commented:
Package coredns - 0.2.1 containing this change is available at https://epr.elastic.co/search?package=coredns

elasticmachine commented:
Package couchbase - 0.13.1 containing this change is available at https://epr.elastic.co/search?package=couchbase

elasticmachine commented:
Package couchdb - 0.3.1 containing this change is available at https://epr.elastic.co/search?package=couchdb

elasticmachine commented:
Package crowdstrike - 1.10.1 containing this change is available at https://epr.elastic.co/search?package=crowdstrike

elasticmachine commented:
Package cyberark_pta - 0.4.1 containing this change is available at https://epr.elastic.co/search?package=cyberark_pta

elasticmachine commented:
Package cyberarkpas - 2.9.1 containing this change is available at https://epr.elastic.co/search?package=cyberarkpas

elasticmachine commented:
Package cylance - 0.12.1 containing this change is available at https://epr.elastic.co/search?package=cylance

elasticmachine commented:
Package darktrace - 0.3.1 containing this change is available at https://epr.elastic.co/search?package=darktrace

elasticmachine commented:
Package ded - 1.0.1 containing this change is available at https://epr.elastic.co/search?package=ded

elasticmachine commented:
Package dga - 0.0.5 containing this change is available at https://epr.elastic.co/search?package=dga

elasticmachine commented:
Package docker - 2.4.2 containing this change is available at https://epr.elastic.co/search?package=docker

elasticmachine commented:
Package etcd - 0.3.1 containing this change is available at https://epr.elastic.co/search?package=etcd

elasticmachine commented:
Package f5 - 0.13.1 containing this change is available at https://epr.elastic.co/search?package=f5

elasticmachine commented:
Package fim - 1.4.3 containing this change is available at https://epr.elastic.co/search?package=fim

elasticmachine commented:
Package fireeye - 1.9.1 containing this change is available at https://epr.elastic.co/search?package=fireeye

elasticmachine commented:
Package fortinet_forticlient - 1.3.1 containing this change is available at https://epr.elastic.co/search?package=fortinet_forticlient

elasticmachine commented:
Package fortinet_fortiedr - 1.4.1 containing this change is available at https://epr.elastic.co/search?package=fortinet_fortiedr

elasticmachine commented:
Package fortinet_fortigate - 1.6.1 containing this change is available at https://epr.elastic.co/search?package=fortinet_fortigate

elasticmachine commented:
Package fortinet_fortimail - 1.3.1 containing this change is available at https://epr.elastic.co/search?package=fortinet_fortimail

elasticmachine commented:
Package fortinet_fortimanager - 1.3.1 containing this change is available at https://epr.elastic.co/search?package=fortinet_fortimanager

elasticmachine commented:
Package gcp - 2.17.1 containing this change is available at https://epr.elastic.co/search?package=gcp

elasticmachine commented:
Package github - 1.8.2 containing this change is available at https://epr.elastic.co/search?package=github

elasticmachine commented:
Package google_workspace - 2.3.1 containing this change is available at https://epr.elastic.co/search?package=google_workspace

elasticmachine commented:
Package hadoop - 0.4.2 containing this change is available at https://epr.elastic.co/search?package=hadoop

elasticmachine commented:
Package haproxy - 1.5.1 containing this change is available at https://epr.elastic.co/search?package=haproxy

elasticmachine commented:
Package hashicorp_vault - 1.9.1 containing this change is available at https://epr.elastic.co/search?package=hashicorp_vault

elasticmachine commented:
Package hid_bravura_monitor - 1.5.1 containing this change is available at https://epr.elastic.co/search?package=hid_bravura_monitor

elasticmachine commented:
Package http_endpoint - 1.6.1 containing this change is available at https://epr.elastic.co/search?package=http_endpoint

elasticmachine commented:
Package httpjson - 1.7.1 containing this change is available at https://epr.elastic.co/search?package=httpjson

elasticmachine commented:
Package ibmmq - 0.3.1 containing this change is available at https://epr.elastic.co/search?package=ibmmq

elasticmachine commented:
Package iis - 1.5.1 containing this change is available at https://epr.elastic.co/search?package=iis

elasticmachine commented:
Package influxdb - 0.2.1 containing this change is available at https://epr.elastic.co/search?package=influxdb

elasticmachine commented:
Package infoblox_bloxone_ddi - 0.3.1 containing this change is available at https://epr.elastic.co/search?package=infoblox_bloxone_ddi

elasticmachine commented:
Package infoblox_nios - 1.6.2 containing this change is available at https://epr.elastic.co/search?package=infoblox_nios

elasticmachine commented:
Package journald - 0.0.5 containing this change is available at https://epr.elastic.co/search?package=journald

elasticmachine commented:
Package juniper_netscreen - 0.6.1 containing this change is available at https://epr.elastic.co/search?package=juniper_netscreen

elasticmachine commented:
Package juniper_srx - 1.9.1 containing this change is available at https://epr.elastic.co/search?package=juniper_srx

elasticmachine commented:
Package kafka - 1.4.1 containing this change is available at https://epr.elastic.co/search?package=kafka

elasticmachine commented:
Package kafka_log - 1.1.1 containing this change is available at https://epr.elastic.co/search?package=kafka_log

elasticmachine commented:
Package kubernetes - 1.32.1 containing this change is available at https://epr.elastic.co/search?package=kubernetes

elasticmachine commented:
Package lastpass - 0.3.1 containing this change is available at https://epr.elastic.co/search?package=lastpass

elasticmachine commented:
Package lyve_cloud - 1.0.2 containing this change is available at https://epr.elastic.co/search?package=lyve_cloud

elasticmachine commented:
Package m365_defender - 1.6.2 containing this change is available at https://epr.elastic.co/search?package=m365_defender

elasticmachine commented:
Package microsoft_defender_endpoint - 2.8.2 containing this change is available at https://epr.elastic.co/search?package=microsoft_defender_endpoint

elasticmachine commented:
Package microsoft_dhcp - 1.10.1 containing this change is available at https://epr.elastic.co/search?package=microsoft_dhcp

elasticmachine commented:
Package microsoft_exchange_online_message_trace - 0.4.1 containing this change is available at https://epr.elastic.co/search?package=microsoft_exchange_online_message_trace

elasticmachine commented:
Package microsoft_sqlserver - 1.13.1 containing this change is available at https://epr.elastic.co/search?package=microsoft_sqlserver

elasticmachine commented:
Package mimecast - 1.6.5 containing this change is available at https://epr.elastic.co/search?package=mimecast

elasticmachine commented:
Package modsecurity - 1.6.1 containing this change is available at https://epr.elastic.co/search?package=modsecurity

elasticmachine commented:
Package mongodb - 1.6.1 containing this change is available at https://epr.elastic.co/search?package=mongodb

elasticmachine commented:
Package mysql_enterprise - 1.4.1 containing this change is available at https://epr.elastic.co/search?package=mysql_enterprise

elasticmachine commented:
Package nagios_xi - 0.3.1 containing this change is available at https://epr.elastic.co/search?package=nagios_xi

elasticmachine commented:
Package nats - 1.3.1 containing this change is available at https://epr.elastic.co/search?package=nats

elasticmachine commented:
Package netscout - 0.12.1 containing this change is available at https://epr.elastic.co/search?package=netscout

elasticmachine commented:
Package netskope - 1.6.1 containing this change is available at https://epr.elastic.co/search?package=netskope

elasticmachine commented:
Package network_traffic - 1.9.3 containing this change is available at https://epr.elastic.co/search?package=network_traffic

elasticmachine commented:
Package nginx - 1.7.1 containing this change is available at https://epr.elastic.co/search?package=nginx

elasticmachine commented:
Package o365 - 1.13.1 containing this change is available at https://epr.elastic.co/search?package=o365

elasticmachine commented:
Package okta - 1.16.1 containing this change is available at https://epr.elastic.co/search?package=okta

elasticmachine commented:
Package oracle - 1.11.1 containing this change is available at https://epr.elastic.co/search?package=oracle

elasticmachine commented:
Package osquery - 1.7.1 containing this change is available at https://epr.elastic.co/search?package=osquery

elasticmachine commented:
Package panw - 3.5.2 containing this change is available at https://epr.elastic.co/search?package=panw

elasticmachine commented:
Package panw_cortex_xdr - 1.7.1 containing this change is available at https://epr.elastic.co/search?package=panw_cortex_xdr

elasticmachine commented:
Package pfsense - 1.6.3 containing this change is available at https://epr.elastic.co/search?package=pfsense

elasticmachine commented:
Package php_fpm - 0.2.2 containing this change is available at https://epr.elastic.co/search?package=php_fpm

elasticmachine commented:
Package platform_observability - 0.0.2 containing this change is available at https://epr.elastic.co/search?package=platform_observability

elasticmachine commented:
Package postgresql - 1.6.1 containing this change is available at https://epr.elastic.co/search?package=postgresql

elasticmachine commented:
Package prometheus - 1.3.1 containing this change is available at https://epr.elastic.co/search?package=prometheus

elasticmachine commented:
Package redisenterprise - 0.3.1 containing this change is available at https://epr.elastic.co/search?package=redisenterprise

elasticmachine commented:
Package santa - 3.5.1 containing this change is available at https://epr.elastic.co/search?package=santa

elasticmachine commented:
Package sentinel_one - 1.5.2 containing this change is available at https://epr.elastic.co/search?package=sentinel_one

elasticmachine commented:
Package snort - 1.4.2 containing this change is available at https://epr.elastic.co/search?package=snort

elasticmachine commented:
Package sonicwall_firewall - 1.3.2 containing this change is available at https://epr.elastic.co/search?package=sonicwall_firewall

elasticmachine commented:
Package sophos - 2.7.1 containing this change is available at https://epr.elastic.co/search?package=sophos

elasticmachine commented:
Package squid - 0.12.1 containing this change is available at https://epr.elastic.co/search?package=squid

elasticmachine commented:
Package stan - 1.3.1 containing this change is available at https://epr.elastic.co/search?package=stan

elasticmachine commented:
Package statsd_input - 0.0.2 containing this change is available at https://epr.elastic.co/search?package=statsd_input

elasticmachine commented:
Package suricata - 2.6.1 containing this change is available at https://epr.elastic.co/search?package=suricata

elasticmachine commented:
Package synthetics - 0.11.7 containing this change is available at https://epr.elastic.co/search?package=synthetics

elasticmachine commented:
Package system - 1.25.1 containing this change is available at https://epr.elastic.co/search?package=system

elasticmachine commented:
Package tcp - 1.7.1 containing this change is available at https://epr.elastic.co/search?package=tcp

elasticmachine commented:
Package tomcat - 1.8.1 containing this change is available at https://epr.elastic.co/search?package=tomcat

elasticmachine commented:
Package trend_micro_vision_one - 0.3.1 containing this change is available at https://epr.elastic.co/search?package=trend_micro_vision_one

elasticmachine commented:
Package udp - 1.7.1 containing this change is available at https://epr.elastic.co/search?package=udp

elasticmachine commented:
Package vsphere - 1.3.2 containing this change is available at https://epr.elastic.co/search?package=vsphere

elasticmachine commented:
Package websphere_application_server - 0.3.1 containing this change is available at https://epr.elastic.co/search?package=websphere_application_server

elasticmachine commented:
Package winlog - 1.12.3 containing this change is available at https://epr.elastic.co/search?package=winlog

elasticmachine commented:
Package zeek - 2.7.1 containing this change is available at https://epr.elastic.co/search?package=zeek

elasticmachine commented:
Package zerofox - 1.7.1 containing this change is available at https://epr.elastic.co/search?package=zerofox

elasticmachine commented:
Package zookeeper - 1.5.1 containing this change is available at https://epr.elastic.co/search?package=zookeeper

elasticmachine commented:
Package zoom - 1.7.1 containing this change is available at https://epr.elastic.co/search?package=zoom

elasticmachine commented:
Package zscaler_zpa - 1.5.2 containing this change is available at https://epr.elastic.co/search?package=zscaler_zpa

elasticmachine commented:
Package google_workspace - 2.4.0 containing this change is available at https://epr.elastic.co/search?package=google_workspace

elasticmachine commented:
Package cloud_security_posture - 1.3.0-preview4 containing this change is available at https://epr.elastic.co/search?package=cloud_security_posture

elasticmachine commented:
Package cloud_security_posture - 1.3.0-preview5 containing this change is available at https://epr.elastic.co/search?package=cloud_security_posture

elasticmachine commented:
Package cloud_security_posture - 1.3.0-preview6 containing this change is available at https://epr.elastic.co/search?package=cloud_security_posture

elasticmachine commented:
Package cloud_security_posture - 1.3.0-preview7 containing this change is available at https://epr.elastic.co/search?package=cloud_security_posture

elasticmachine commented:
Package cloud_security_posture - 1.3.0-preview8 containing this change is available at https://epr.elastic.co/search?package=cloud_security_posture

elasticmachine commented:
Package cloud_security_posture - 1.3.0-preview9 containing this change is available at https://epr.elastic.co/search?package=cloud_security_posture

elasticmachine commented:
Package cloud_security_posture - 1.3.0-preview10 containing this change is available at https://epr.elastic.co/search?package=cloud_security_posture

elasticmachine commented:
Package cloud_security_posture - 1.3.0-preview91 containing this change is available at https://epr.elastic.co/search?package=cloud_security_posture

elasticmachine commented:
Package cloud_security_posture - 1.3.1 containing this change is available at https://epr.elastic.co/search?package=cloud_security_posture
